shared/versions.yml

```diff
@@ -63,19 +63,27 @@ couchdb:
     tag: 2.3.0
 dataloader:
   upstream:
-    repository: gpii/universal
+    repository: herrclown/vagrant-universal
```

Comment:
Reminder that you'll need to fix up these repositories before merging.

Reply:
More specifically, when [PR #777](https://github.com/GPII/universal/pull/777) for GPII-3646 is merged, this can be set to `gpii/universal`

Reply:
Actually, it occurs to me that only the `gpii-flushtokens` component (module?) needs the herrclown universal image.  I'll change things to show just that dependency for now.